removing backup_storage_hook from the doc (#1366)
vitessio/website: content/en/docs/16.0/reference/programs/vtbackup.md

## Example Usage

> Consider you have running Vitess cluster, the following command will create backup using `vtbackup` for keyspace `commerce` having shard `0`

Reviewer: ```suggestion
On a running Vitess cluster, the following command will create a backup using `vtbackup` for keyspace `commerce` and shard `0`.
```

Reviewer: The previous version didn't sound right when I read it out.
